ablySpec/Auth.swift

```diff
@@ -177,34 +177,66 @@ class Auth : QuickSpec {
             // RSA15
             context("token auth and clientId") {
                 // RSA15a
-                it("should check clientId consistency") {
-                    let expectedClientId = "client_string"
-                    let options = AblyTests.setupOptions(AblyTests.jsonRestOptions)
-                    options.clientId = expectedClientId
-                    
-                    let client = ARTRest(options: options)
-                    client.httpExecutor = mockExecutor
-                    
-                    waitUntil(timeout: 10) { done in
-                        // Token
-                        client.calculateAuthorization(ARTAuthMethod.Token) { token, error in
-                            if let e = error {
-                                XCTFail(e.description)
+                context("should check clientId consistency") {
+
+                    it("on rest") {
+                        let expectedClientId = "client_string"
+                        let options = AblyTests.setupOptions(AblyTests.jsonRestOptions)
+                        options.clientId = expectedClientId
+
+                        let client = ARTRest(options: options)
+                        client.httpExecutor = mockExecutor
+
+                        waitUntil(timeout: 10) { done in
+                            // Token
+                            client.calculateAuthorization(ARTAuthMethod.Token) { token, error in
```

Comment:
@mattheworiordan It is the REST method that checks the Auth method and completes the request with the Authorisation header: `Basic key` or `Bearer token`.